src/main/java/io/cdap/plugin/kudu/KuduSink.java

```diff
@@ -188,7 +191,7 @@ public void destroy() {
   /**
    * Creates a Kudu table if it doesn't exist.
    */
-  private void createKuduTable() {
+  private void createKuduTable(FailureCollector failureCollector) {
```

Comment:
this no longer needs to take a failure collector, as configurePipeline() should not be calling it.

Reply:
Done.